src/aind_data_transfer/transformations/metadata_creation.py

```diff
@@ -351,9 +355,11 @@ def from_inputs(
             if isinstance(funding_source, tuple)
             else funding_source
         )
-        data_description_instance = RawDataDescription.from_name(
-            name,
+        basic_settings = RawDataDescription.parse_name(name=name)
```

Comment:
We can add `from_name` back if you want.

Reply:
I think it should be fine without any changes.